src/features/areas/utils/getMarkerValues.tsx

```diff
@@ -0,0 +1,20 @@
+import { Map } from 'leaflet';
+
+type Marker = {
+  markerX: number;
+  markerY: number;
+};
+
+const getMarkerValues = (map: Map, crosshair: HTMLDivElement): Marker => {
```

Comment:
I'm thinking this could have a better name, something like `getCrosshairPositionOnMap()` or something. What do you think?

I'm honestly not sure what this function does exactly, so I think whatever it's new name is needs to be decided based on a good understanding of what the function does, and better convey what it does.

Reply:
Unfortunately I don't understand 100% what the function does too, but I agree that `getCrosshairPositionOnMap()` it's a better name so I'll change it to that for now.